N/normaliz/build_tarballs.jl

```diff
@@ -37,6 +37,7 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency("GMP_jll", v"6.1.2"),
+    BuildDependency(PackageSpec(name="MPFR_jll", version=v"4.0.2")),
```

Comment:
Actually, MPFR *is* a dependency of libnormaliz, just one that was forgotten to declare. So I'd suggest this:
```suggestion
    Dependency("MPFR_jll", v"4.0.2"),
```

Reply:
No it is not, it is only needed for FLINT:
```
$ grep -ir mpfr source/
source/Makefile.configuration:FLINTFLAGS = -L $(OPT_LIB_ROOT)/lib -lflint -L $(OPT_LIB_ROOT)/lib -lmpfr
source/Makefile.configuration:  EANTICFLAGS = -L $(OPT_LIB_ROOT)/lib -leanticxx -leantic -larb -lflint -lmpfr
```

Reply:
What do you mean? As you just confirmed it does link against libmpfr, as one can also verify in the generated binaries:
```
$ otool -L ~/.julia/artifacts/*/lib/libnormaliz.dylib
/Users/mhorn/.julia/artifacts/34750e7597595ecce93a0a04cd79cbb7514e475f/lib/libnormaliz.dylib:
        @rpath/libnormaliz.3.dylib (compatibility version 12.0.0, current version 12.5.0)
        @rpath/libflint-14.dylib (compatibility version 14.0.0, current version 14.0.3)
        @rpath/libmpfr.6.dylib (compatibility version 8.0.0, current version 8.0.0)
        @rpath/libnauty.2.dylib (compatibility version 9.0.0, current version 9.12.0)
        @rpath/libgmpxx.4.dylib (compatibility version 10.0.0, current version 10.2.0)
        @rpath/libgmp.10.dylib (compatibility version 14.0.0, current version 14.2.0)
        /usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 307.4.0)
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1238.0.0)
/Users/mhorn/.julia/artifacts/74382a30a993ada2364b4247850ebfba13c3b86a/lib/libnormaliz.dylib:
        @rpath/libnormaliz.3.dylib (compatibility version 12.0.0, current version 12.5.0)
        @rpath/libflint-14.dylib (compatibility version 14.0.0, current version 14.0.0)
        @rpath/libmpfr.6.dylib (compatibility version 7.0.0, current version 7.2.0)
        @rpath/libnauty.2.dylib (compatibility version 9.0.0, current version 9.12.0)
        @rpath/libgmpxx.4.dylib (compatibility version 10.0.0, current version 10.2.0)
        @rpath/libgmp.10.dylib (compatibility version 14.0.0, current version 14.2.0)
        /usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 120.0.0)
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1213.0.0)
/Users/mhorn/.julia/artifacts/cb0dd229ec1ce21154b81ebc8ab59675664cc2eb/lib/libnormaliz.dylib:
        @rpath/libnormaliz.3.dylib (compatibility version 12.0.0, current version 12.5.0)
        @rpath/libflint-14.dylib (compatibility version 14.0.0, current version 14.0.0)
        @rpath/libmpfr.6.dylib (compatibility version 7.0.0, current version 7.2.0)
        @rpath/libnauty.2.dylib (compatibility version 9.0.0, current version 9.12.0)
        @rpath/libgmpxx.4.dylib (compatibility version 10.0.0, current version 10.2.0)
        @rpath/libgmp.10.dylib (compatibility version 14.0.0, current version 14.2.0)
        /usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 120.0.0)
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1213.0.0)
```

Of course currently this distinction is moot, as MPFR_jll gets pulled in as an indirect dependency.

Reply:
Just to be clear, this is a minor quibble and I hope that someone will merge this PR soon :-)

Reply:
The grep was meant to show that there is no code at all referring to mpfr.

That one needs to specify all dependencies (even indirect ones) when linking is just the way shared libraries work, in many cases one would get this `-lmpfr` flag from e.g. `pkg-config --libs` instead of hardcoding it.